src/screens/SecretIconsScreen.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import { FlatList, StyleSheet, Text, View } from 'react-native';
+
+import { RegularText } from '../components';
+import { colors, Icon, normalize } from '../config';
+
+type TIconItem = {
+  iconName: string;
+};
+
+const iconNames = Object.keys(Icon);
+
+const IconItem = ({ iconName }: TIconItem) => {
+  const SelectedIcon = Icon[iconName];
+  return (
```

Comment:
I recommend leaving a line space between SelectedIcon and return section.

Reply:
Adjusted it!